src/sentry/utils/eventuser.py

```diff
@@ -316,7 +318,28 @@ def for_tags(cls, project_id: int, values: list[str]) -> dict[str, EventUser]:
         Return a dictionary of {tag_value: event_user}.
         """
         projects = Project.objects.filter(id=project_id)
+        result = {}
+
+        # If we have too many values, batch them to avoid query size limits
+        if len(values) <= MAX_TAG_VALUES_BATCH_SIZE:
```

Comment:
nit: The comment says "if _condition_", but then the code checks "if _not condition_" which confused me at first glance, I think this comment makes more sense above the for block.